### PR TITLE
Added Projects In Readme

### PR DESCRIPTION
**Pull request by pranjay-poddar**
### 👨‍💻 Changes proposed
Added Projects In Readme.

### ✅ Check List (Check all the applicable boxes) 

- [X] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [X] This PR does not contain plagiarized content.
- [X] My Addition/Changes works properly and matches the overall repo pattern.
- [X] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,7 +247,7 @@ git push -u origin <your_branch_name>
...
...blic/ReportCalculator.html)| 120. | [Responsive Landing](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/responsiveLanding.html)| 121. | [Reviews](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/reviews.html)| 122. | [Snake](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/snake.html)| 123. | [Socials](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/socials.html)| 124. | [Star Catching Game](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/star-catching-game.html)| 125. | [Sticky Notes](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/sticky_notes.html)| 126. | [Stopwatch](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/stopwatch.html)| 127. | [Tap Music](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/tap_music.html)| 128. | [Tiktok Clone](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/Tiktok_Clone.html)| 129. | [Toast Notification](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/toast_notification.html)| 130. | [To Do](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/todo.html)
```
Comment (Astrodevil) [inline README.md:250]: Your changes are not properly reflected. You can see the whole new addition appears in just line 250. Please change lines after every 3 addition.

Comment (pranjay-poddar) [reply README.md]: Made the required changes, @Astrodevil hope it's in the correct format now.

**Comment by pranjay-poddar** [comment]
Made all the required changes hope they are in correct format.

**Comment by Astrodevil** [comment]
- 109 is already added in 9
- 127 is already added in 12

But leave as it is, New projects can be added at that place.

**Pull request merged**